homeassistant/const.py

```diff
@@ -108,6 +108,7 @@
 CONF_PENDING_TIME = 'pending_time'
 CONF_PIN = 'pin'
 CONF_PLATFORM = 'platform'
+CONF_PROCEED = 'proceed'
```

Comment:
`proceed` is not self-explain to me. How about `continue_on_timeout` 

And it should resident in `helpers/script.py` together with `CONF_WAIT_TEMPLATE`.